charts/k8s-monitoring/values.yaml

```
@@ -389,6 +389,9 @@ logs:
     # -- Capture and forward logs from Kubernetes pods
     enabled: true
 
+    # -- Only capture logs from pods in these namespaces (empty means all namespaces)
```

```suggestion
    # -- Only capture logs from pods in these namespaces (`[]` means all namespaces)
```

Wonder if this is worth clarifying? and if helm docs supports markdown formatting... same for line 413.